### DISCUSSION TITLE
Unity 2020.3.2f1 Doesn't recieve messages from react

### DISCUSSION THREAD

**Discussion by MatteoVilardo** [jeffreylanters/react-unity-webgl#174] [Help Wanted!]
i am using your react library to interact with my unity application, but also with a simple test is not working, i suppose is a compatibility issue.

this is the line of code to send the message to unity : unityContent.send(“MyObject”,”TestMethod”,true);
of course in unity there is a gameobject called “MyObject” and a script with a public method called TestMethod(bool value);

The react library version is the latest (8.3.1) and i can see the page with my application, but i cannot interact it trough react.

// React code
unityContent.send("MyObject","TestMethod",true);

C# Code
`public class TestReact : MonoBehaviour 
{
     public void TestMethod(bool test) 
     {
          Debug.Log($"Test message {test}"); 
         Camera.main.backgroundColor = Color.blue; 
      }
 }`

**Comment by jeffreylanters** [comment]
Hi! Is your `TestReact` MonoBehaviour attached to a GameObject named exactly `MyObject`, and is this GameObject enabled and active in the hierarchy when invoking the send method onto your unityContext? I also noticed you've wrote `unityContent.send...`, but there is no such class as `unityContent`, only `unityContext`. But it is just a variable name of yours, is this a typo? Did you import and instantiate the correct class? What error are you getting?

**Reply by MatteoVilardo** [reply]
Yes, unityContent is just a variable name, the type is UnityContext, and i don't recieve any error, the unity part is properly arranged. I can visualize correctly the unity scene into my react page, but i can't send message

**Reply by MatteoVilardo** [reply]
![image](https://user-images.githubusercontent.com/25264917/113406654-23d8c980-93ac-11eb-8d58-7c6c2ed49df4.png)

I hope this can help you

**Reply by jeffreylanters** [reply]
That errors seems to be coming from your unity build itself. Does the unity application start at all?

**Reply by MatteoVilardo** [reply]
Yeah i resolved the error, which was caused by the unityContext.on("Debug") instead of unityContext.on("debug"), but my main issue still remain, i cannot invoke a method from javascript to unity

**Reply by jeffreylanters** [reply]
Can you compare your code to the [test environment](https://github.com/jeffreylanters/react-unity-webgl-test) to see if there is anything you've missed?

**Reply by AditiChauhan12** [reply]
I'm facing the same issue. Has it been resolved? What's the solution?

**Reply by MatteoVilardo** [reply]
Hi, yeah i have resolved but it was a year ago, and i don't remember what i have done if i can help you i can send it a little script which the role is communicate between unity and js

**Reply by AditiChauhan12** [reply]
Thanks! I figured it out. I was missing the StreamingAssets path.
